### PR TITLE
Api

### PR DESCRIPTION
**Pull request by kmmraj**
Added API to call protractor-perf from grunt. Added sample grunt file and explanation in Readme.

### CODE REVIEW

**Comment by axemclion** [comment]
@kmmraj Thanks for this - a lot of folks have been asking for the integration -https://github.com/axemclion/protractor-perf/issues/1

Can I request you to merge all the commits into a single commit - will be easier to consume them.

File: sampleGruntFile.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = function(grunt) {
+
+	 var protractorperf = require('protractor-perf');
```
Comment (JuhQ) [inline sampleGruntFile.js:3]: Can you fix the indentation from this file? Seems inconsistent.

**Comment by JuhQ** [comment]
@kmmraj why did you close the pr? :(

**Comment by kmmraj** [comment]
I have opened a new PR.

**Pull request closed without merging**